cygal/src/routes/index.svelte

```diff
@@ -1,4 +1,4 @@
-<h1>Welcome to SvelteKit</h1>
+<h1 class="text-4xl underline decoration-rose-400">Welcome to SvelteKit</h1>
```

Comment:
Wauw such a mindful test!

![test](https://c.tenor.com/zgL4vqoPcugAAAAC/buddha-relax.gif)

Reply:
no outstanding issue by the way.

Reply:
thank you! I removed the package-lock.json file at the root (kept the other one in /cygal because we need it for the pre-commit check for caching)